great-idea/css/index.css

```diff
@@ -63,4 +58,177 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	width: 876px;
+	margin: 0 auto;
+}
+
+.navigation {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	width: 876px;
+	margin-top: 25px;
+}
+
+.navigation a {
+	text-decoration: none;
+	color: grey;
+}
+
+header {
+	height: 450px;
+	text-align: center;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	margin-top: 25px;
+}
+
+.h1-div {
+	font-size: 75px;
+	width: 290px;
+}
+
+button {
+	text-decoration: none;
+	border: 1px solid black;
+	background: white;
+	font-size: 15px;
+	width: 140px;
+	height: 30px;
+}
+
+.top-content {
+	border-top: 2px solid black;
+	text-align: justify;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	display: flex;
+	justify-content: space-between;
+}
+
+.top-content div {
+	width: 400px;
+}
+
+.bottom-content {
+	text-align: left;
+	border-bottom: 2px solid black;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	display: flex;
```

Comment:
flexbox has some default properties that tend to take over but you can hard code it in there just to be sure that it is doing what you want it. In this case I like to always manually set the direction and in some cases if I want it to wrap or not. This will make things easier for you when you get into responsive design as you don't want your elements behavior to be automatically decided by flexbox all the time.